src/components/FarmTable/index.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react'
+import { Flex } from 'rebass'
+import styled from 'styled-components'
+import Loader from '../Loaders/table'
+import { Rewards } from '../../utils/farm/constants'
+import Filter from './filter'
+import Reward from './reward'
+
+const Wrap = styled('div')`
+  margin-bottom: 25px;
+`
+
+const Table = styled('div')`
+  background: #232638;
+  border-radius: 16px;
+  font-size: 16px;
+  font-weight: 500;
+  text-align: center;
+`
+const Header = styled('div')`
+  display: flex;
+  flex: wrap;
+  font-size: 16px;
+`
+
+const Column = styled(Flex)`
+  border-bottom: 1px solid black;
+  padding-left: 25px;
+  text-align: left;
+  line-height: 3rem;
+`
+
+export default function FarmTable({ rewards }: Rewards) {
+  const [filter, setFilter] = useState<boolean>(true)
```

Comment:
it seems you haven't hooked up the filter, you are passing` true` on line 38 and 55 instead of the `filter` variable